configure

```diff
@@ -796,6 +801,8 @@ def configure_v8(o):
   o['variables']['v8_optimized_debug'] = 0  # Compile with -O0 in debug builds.
   o['variables']['v8_random_seed'] = 0  # Use a random seed for hash tables.
   o['variables']['v8_use_snapshot'] = 'false' if options.without_snapshot else 'true'
+  if options.with_v8_tests:
+    o['variables']['with_v8_tests'] = 1
```

Comment:
Are the changes to `configure` necessary?  `with_v8_tests` doesn't seem to be used anywhere.